static/js/stripe-button.js

```diff
@@ -0,0 +1,42 @@
+$(document).ready(function(){
+
+  $.getScript("https://checkout.stripe.com/checkout.js", function()
+  {
+    console.log("stripe checkout script ready");
+
+    var handler = StripeCheckout.configure({
+      key: 'pk_test_mlBqEQWc5FR1hGVCPuBH1kgy',
+      image: 'https://stripe.com/img/documentation/checkout/marketplace.png',
+      locale: 'auto',
+      token: function(token) {
+    // You can access the token ID with `token.id`.
+    // Get the token ID to your server-side code for use.
+    $("#stripeToken").val(token.id);
+    $("#stripeEmail").val(token.email);
+    $("#stripe-form").submit();
+      }
+    });
+
+    $('#stripe-button').on('click', function (e) {
+    // Open Checkout with further options
+    handler.open({
+      name: 'ARM New Brunswick',
+      description: 'armnewbrunswick.org',
+      amount: 100,
```

Comment:
Just a quick question, so the `amount` param here is the suggested donation amount in the form yes?